tagstory/src/main/java/no/tagstory/utils/StoryParser.java

```diff
@@ -174,14 +194,34 @@ private static void parseGameMode(StoryTag storyTag, JSONObject jsonTag) throws
 			storyTag.initQuizMode();
 			JSONArray quiz = jsonTag.getJSONArray(QUIZ);
 			JSONObject question;
+			ArrayList<String> multiQuizAnswers = new ArrayList<String>();
+			int location;
+			String quizKey;
+			//TODO, add a list of answers for quiz
 
 			for (int index = 0; index < quiz.length(); index++) {
 				question = quiz.getJSONObject(index);
+				checkQuizType(index,storyTag,jsonTag,question, multiQuizAnswers);
+			}
+			quizCounter = 1;
```

Comment:
Hvorfor bruke qui quizcounter, og ikke bare indexen din i for-løkka?
Kan jo bare sende den inn i checkQuizType.